packages/manager/core/api/src/iceberg.ts

```diff
@@ -92,10 +92,10 @@ export const buildHeaders = () => {
       if (filters?.length) {
         const filtersJoin = filters
           .filter(({ type }) => type !== FilterTypeCategories.Tags)
-          .map(
-            ({ comparator, key, value }) =>
-              `${encodeURIComponent(key)}:${icebergFilter(comparator, String(value || ''))}`,
-          )
+          .map(({ comparator, key, value }) => {
+            const correctedValue = typeof value === 'object' ? value : String(value || '');
+            return `${encodeURIComponent(key)}:${icebergFilter(comparator, correctedValue)}`;
+          })
```

Comment:
Actually this is not the right way to fix the issue. According to Iceberg documentation, the filter with `isIn` needs to be `filedName:in=val1,val2`. So the actual fix will result in an Iceberg API call failure.